Data/src/main/java/org/tribuo/data/columnar/processors/response/BinaryResponseProcessor.java

```diff
@@ -47,6 +52,36 @@
     @Config(description="The negative response to emit.")
     private String negativeName = "0";
 
+    @Config(description = "A list of field names to read, you should use only one of this or fieldName.")
+    private List<String> fieldNames;
+
+    @Config(description = "A list of strings that trigger positive responses; it should be the same length as fieldNames or empty")
+    private List<String> positiveResponses;
+
+    @Config(description = "Whether to display field names as part of the generated label, defaults to false")
+    private boolean displayField = false;
+
+
+    @Override
+    public void postConfig() {
+        if (fieldName != null && fieldNames != null) { // we can only have one path
+            throw new PropertyException("fieldName, FieldNames", "only one of fieldName or fieldNames can be populated");
+        } else if (fieldNames != null) {
+            positiveResponses = positiveResponses == null ? Collections.nCopies(fieldNames.size(), positiveResponse) : positiveResponses;
+            if(positiveResponses.size() != fieldNames.size()) {
+                throw new PropertyException("positiveResponses", "must either be empty or match the length of fieldNames");
+            }
+        } else if (fieldName != null)  {
+            if(positiveResponses != null) {
+                throw new PropertyException("positiveResponses", "if fieldName is populated, positiveResponses must be blank");
+            }
+            fieldNames = Collections.singletonList(fieldName);
+            positiveResponses = Collections.singletonList(positiveName);
```

Comment:
This is `positiveName` not `positiveResponse`? Also it should check that it's not null.